api/controllers/v0.2/createPatron.js

```diff
@@ -198,17 +200,17 @@ function callAxiosToCreatePatron(req, res) {
           renderResponse(req, res, 201, modeledResponse); // respond with 201 even if streaming fails
         });
     })
-    .catch((axiosErrorResponse) => {
+    .catch((axiosError) => {
       try {
```

Comment:
Let's insert a line outputting to log.

Reply:
Fortunately collectErrorResponseData sends output to the log, and it is called a few lines down.